src/index.jsx

```diff
@@ -0,0 +1,10 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import App from './components/App/App.jsx';
+
+const root = <div id="root">1</div>;
```

Comment:
See suggestion below:

```suggestion
const rootElement = document.createElement('div');
rootElement.setAttribute('id', 'root');
document.body.appendChild(rootElement);
```